packages/mobile/components/AddressBox.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="typescript">
+    import { removeDisplayNotification, showAppNotification } from '@auxiliary/notification'
+    import { localize } from '@core/i18n'
+    import { setClipboard } from '@core/utils'
+    import { Box, FontWeight, Text, TextType } from 'shared/components'
+
+    export let address = ''
+    export let fontSize = 'base'
+
+    let notificationId
+
+    export function copyAddress(): void {
```

Comment:
my bad not updating the requirements of the task, I dont think we need this, we can keep the current behavior and remove this